src/Orleans/Streams/PersistentStreams/IPersistentStreamPullingAgent.cs

```diff
@@ -13,7 +13,7 @@ internal interface IPersistentStreamPullingAgent : ISystemTarget, IStreamProduce
 
     internal interface IPersistentStreamPullingManager : ISystemTarget
     {
-        Task Initialize(Immutable<IQueueAdapter> queueAdapter);
+        Task Initialize(Immutable<IQueueAdapter> queueAdapter, Immutable<IStreamQueueMapper> queueMapper);
```

Comment:
U are fast ! wait for 30 mins and come back ! I just submitted this. haven't clean up yet.